impexp-client/src/main/java/org/citydb/modules/kml/gui/preferences/GeneralPanel.java

```diff
@@ -161,6 +167,8 @@ private void initGui(ViewController viewController) {
 		createGltfCheckbox.setIconTextGap(10);
 		notCreateColladaCheckbox.setIconTextGap(10);
 		embedTexturesInGltfCheckbox.setIconTextGap(10);
+		exportGltfBinary.setIconTextGap(10);
+		enableGltfDracoCompression.setIconTextGap(10);
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).